app/main.py

```diff
@@ -16,4 +16,24 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+unsorted_vars = {
+    "lucky_number": lucky_number,
+    "pi": pi,
+    "one_is_a_prime_number": one_is_a_prime_number,
+    "name": name,
+    "my_favourite_films": my_favourite_films,
+    "profile_info": profile_info,
+    "marks": marks,
+    "collection_of_coins": collection_of_coins,
+}
+sorted_variables = {
+    "mutable": [],
+    "immutable": [],
+}
+mutable_types = [set, list, dict]
+
+for key, value in unsorted_vars.items():
```

Comment:
it seems to me, that u don't need key anywhere

Reply:
> it seems to me, that u don't need key anywhere

I believe I tried different approaches and at some point of time I got different parts of code from different tries so I got this Frankinstain